app/components/publishers/vacancies_component.rb

```diff
@@ -46,10 +46,10 @@ def view_applicants(vacancy, card)
     return unless vacancy.enable_job_applications?
     return unless include_job_applications?
 
-    after_submission_job_applications = vacancy.job_applications.after_submission
+    applications = vacancy.job_applications.where(status: %w[submitted reviewed shortlisted unsuccessful])
```

Comment:
This does not change the pre-existing logic of this method. I changed the definition of the `after_submission` scope in this PR to include withdrawn applications, which had been deliberately excluded here.